test/UnitTests/MSTest.Analyzers.UnitTests/PreferTestCleanupOverDisposeAnalyzerTests.cs

```diff
@@ -44,8 +65,22 @@ public class MyTestClass : IAsyncDisposable
                 }
             }
             """;
+        string fixedCode = """
+            using System;
+            using System.Threading.Tasks;
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+            [TestClass]
+            public class MyTestClass {
```

Comment:
@engyebrahim The formatting looks off here, you are probably missing the trivias from the syntax you changed/removed. Could you please create a ticket for that?